reV/SAM/generation.py

```diff
@@ -228,41 +228,42 @@ def collect_outputs(self, output_lookup=None):
     def _gen_exec(self):
         """Run SAM generation with possibility for follow on econ analysis."""
 
-        lcoe_out_req = None
-        so_out_req = None
+        lcoe_out_reqs = None
+        so_out_reqs = None
         if 'lcoe_fcr' in self.output_request:
-            lcoe_out_req = self.output_request.pop(
-                self.output_request.index('lcoe_fcr'))
+            lcoe_vars = ('lcoe_fcr', 'fixed_charge_rate', 'capital_cost',
+                         'fixed_operating_cost', 'variable_operating_cost')
+            lcoe_out_reqs = [r for r in self.output_request if r in lcoe_vars]
+            self.output_request = [r for r in self.output_request
+                                   if r not in lcoe_out_reqs]
         else:
-            so_reqs = ('ppa_price', 'lcoe_real', 'lcoe_nom')
-            reqs = [r for r in so_reqs if r in self.output_request]
-            if len(reqs) > 1:
-                raise KeyError('Cannot request more than one single owner '
-                               'output in Generation module. Found the '
-                               'following {} single owner output requests in '
-                               'the generation run: {}'
-                               .format(len(reqs), reqs))
-            elif len(reqs) == 1:
-                so_out_req = self.output_request.pop(
-                    self.output_request.index(reqs[0]))
-
+            so_vars = ('ppa_price', 'lcoe_real', 'lcoe_nom',
```

Comment:
Where is single owner turned on?

Reply:
The block of code you commented on is where generation has the possibility for running econ as a sort of tag-along computation. The first block of code checks for SO output requests and the second block actually runs the SO as a tag-along:

https://github.com/NREL/reV/blob/c30a64955934cc4fe3d9e44beb349b9ce43c1165/reV/SAM/generation.py#L242-L245

https://github.com/NREL/reV/blob/c30a64955934cc4fe3d9e44beb349b9ce43c1165/reV/SAM/generation.py#L264-L271